app/indexers/rights_metadata_datastream_indexer.rb

```diff
@@ -62,10 +62,15 @@ def to_solr
       end
     ].flatten.uniq
 
+
```

Comment:
You'll need to remove this line to make the build pass, methinks

Reply:
doh' rubocop

fixed